.gitignore

```diff
@@ -29,8 +29,8 @@ pylint_done
 cprofilerstats*.profile
 /build/
 /build_doc/
-mofparsetab.py
-moflextab.py
+*mofparsetab.py
```

Comment:
I don’t understand the asterisk on this and next line

Reply:
The intention is to exclude both the new name `_mofparsetab.py` and the old name `mofparsetab.py` which can also still exist because an old branch had been checked out and generated the file.

If you are not happy with the asterisk approach to this, we can also specify both file names explicitly. Let me know.

Reply:
Update: I changed this to use two entries for each these two files in `.gitignore`, instead of using wildcards, for better clarity. Also, I added removal of the old files in make clean (since I stumbled across this).
DONE